nodejs/packagescanner.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/quay/zlog"
 
```

Comment:
Should this newline be deleted?

Reply:
Yes, I guess if you're going to reorg imports you should go all the way